ws/Cargo.toml

```diff
@@ -7,7 +7,7 @@ homepage = "https://github.com/paritytech/jsonrpc"
 license = "MIT"
 name = "jsonrpc-ws-server"
 repository = "https://github.com/paritytech/jsonrpc"
-version = "14.2.0"
+version = "14.2.1"
```

Comment:
Or should it be `14.3.0`? cc @ordian

Reply:
I think that's technically a breaking change (so should be `15.0.0`) because of `pub use parity_ws as ws;` (it's also exposed in other public API): `ws 0.9` and `parity-ws 0.10` are two different (semver-incompatible) crates.

Reply:
Yeah, seems the `ws` stuff is exposed for instance in middlewares, which for sure are used by `openethereum` (not necessarily `substrate` though).
Let's bump to `15.0.0` then.

Reply:
> should I bump the version in the PR, and if so, should all crates be bumped to 15.0.0 or just the jsonrpc-ws-server?

Yes and yes. We prefer to keep the major versions synchronized for simplicity even though there weren't any breaking changes.

Reply:
would it make sense to create a `v15.x` branch for that instead of merging it to `v14.x`?

Reply:
I think not; I think the current v15 branch has unrelated and breaking changes. If this here becomes v15, then the current v15-branch will become v16 (correct me if I'm wrong @tomusdrw).

Reply:
@dvdplm could you point me to the `v15` branch? I don't see it listed here https://github.com/paritytech/jsonrpc/branches/all.
Or did you mean `master`? I think `master` can later become `v16` if that's what you mean.